frontend/survey-project/src/app/components/shared/ui/footer/footer.component.ts

```diff
@@ -1,10 +1,24 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { RouteNames } from '../../../../shared/consts/routes'
+import { CookieService } from 'ngx-cookie-service';
 
 @Component({
   selector: 'app-footer',
   templateUrl: './footer.component.html',
   styleUrl: './footer.component.scss'
 })
 export class FooterComponent {
+  RouteNames = RouteNames;
+  hideFooter: boolean = false;
 
+  constructor(private router: Router, private cookieService: CookieService) {
```

Comment:
In the constructor, we should apply new line after each dependency injection we make, to make it more visible.
constructor(
    private router: Router,
    private cookieService: CookieService
) { }